### PR TITLE
Add test coverage for well completions component

### PR DESCRIPTION
**Pull request by aprilzhang**

### CODE REVIEW

File: inputSchema/wellCompletions.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (alexandruandrei91) [inline inputSchema/wellCompletions.json:1]: I guess the location for this file is ok for now, but we probably need to change it if we split the component. We'll probably have to see if we will have it with the component, with the backend or in a separate package.

**Review by alexandruandrei91 — COMMENTED** [review]
When running `npm run verify` there's already 66 warnings. Probably not in this review, but we should try and get the number down since it's becoming hard to see the newly added ones.

File: src/lib/components/WellCompletions/components/Settings/__snapshots__/HideZeroCompletionsSwitch.test.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
[the remark this answers is not in the document]
Comment (aprilzhang) [reply src/lib/components/WellCompletions/components/Settings/__snapshots__/HideZeroCompletionsSwitch.test.tsx.snap:1]: This is a good idea. The snapshots are generated when running the test script in package.json, but .json file doesnt seem to support adding comments. Maybe I could add it somewhere in the developer wiki.

File: inputSchema/wellCompletions.json (hunk 1 above)
Comment (aprilzhang) [reply inputSchema/wellCompletions.json:1]: yes, I agree

File: package.json [hunk 3]
```diff
@@ -59,12 +67,20 @@
...
         "@types/jest": "^26.0.20",
         "@typescript-eslint/eslint-plugin": "^4.19.0",
         "@typescript-eslint/parser": "^4.19.0",
         "ace-builds": "^1.4.12",
+        "addon-redux": "^1.1.0",
         "babel-loader": "^8.1.0",
         "copyfiles": "^2.0.0",
+        "core-js": "^3.9.1",
         "css-loader": "^5.2.0",
         "css-minimizer-webpack-plugin": "^1.3.0",
```
Comment (alexandruandrei91) [inline package.json:85]: This can be removed, since now we're back to optimize-css-..blabla

File: package.json [hunk 4]
```diff
@@ -59,12 +67,20 @@
...
         "cssnano": "^4.1.11",
```
Comment (alexandruandrei91) [inline package.json:86]: You can remove cssnano as an explicit dependency, since that was there for the css minimizer.

File: package.json [hunk 5]
```diff
@@ -75,19 +91,25 @@
...
+        "react-resize-panel": "^0.3.5",
+        "react-test-renderer": "^17.0.1",
+        "redux-mock-store": "^1.5.4",
+        "source-map-loader": "^0.2.4",
         "typescript": "^4.2.3",
         "url-loader": "^4.1.1",
-        "webpack": "^5.28.0",
-        "webpack-cli": "^4.5.0",
-        "webpack-dev-server": "^3.11.2"
+        "webpack": "^4.41.2",
```
Comment (alexandruandrei91) [inline package.json:110]: Maybe create an issue to switch back to webpack5, when Storybook properly supports it.

**Review by alexandruandrei91 — COMMENTED** [review]
Building the storybook creates a folder called `storybook-static` with some files in it. I guess we don't want to commit that, so better put it in .gitignore.

**Review by alexandruandrei91 — APPROVED** [review]
LGTM 👍

**Pull request merged**